sdk/textanalytics/Azure.AI.TextAnalytics/src/ExtractSummaryAction.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// Configurations that allow callers to specify details about how to execute
+    /// an Extract Summary action in a set of documents.
+    /// For example, set the model version, specify the order in which extracted
+    /// sentences are returned, and more.
+    /// </summary>
+    public class ExtractSummaryAction
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ExtractSummaryAction"/>
+        /// class which allows callers to specify details about how to execute
+        /// an Extract Summary action in a set of documents.
+        /// For example, set the model version, specify the order in which extracted
+        /// sentences are returned, and more.
+        /// </summary>
+        public ExtractSummaryAction()
+        {
+        }
+
+        /// <summary>
+        /// Gets or sets a value that, if set, indicates the version of the text
+        /// analytics model that will be used to generate the result.  For supported
+        /// model versions, see operation-specific documentation, for example:
+        /// <see href="https://docs.microsoft.com/azure/cognitive-services/text-analytics/concepts/model-versioning#available-versions"/>.
+        /// </summary>
+        public string ModelVersion { get; set; }
```

Comment:
The service hasn't added a `loggingOptOut` property yet, so we don't have a corresponding `DisableServiceLogs`.

Reply:
Could you create an issue with this? so we don't forget to add it?

Reply:
The most recent version of the swagger should have this now: https://github.com/Azure/azure-rest-api-specs/blob/d758c4205d331c552cafbb755ed02673b9fa5e22/specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.2-preview.1/TextAnalytics.json#L809

Reply:
Thanks for the update. I'll add it in a separate PR since this one is already reviewed and approved. Created https://github.com/Azure/azure-sdk-for-net/issues/22799 for tracking.